pkg/pool-stable/contracts/ComposableStablePool.sol

```diff
@@ -15,7 +15,7 @@
 pragma solidity ^0.7.0;
 pragma experimental ABIEncoderV2;
 
-import "@balancer-labs/v2-interfaces/contracts/pool-stable-phantom/StablePhantomPoolUserData.sol";
+import "@balancer-labs/v2-interfaces/contracts/pool-stable-composable/ComposableStablePoolUserData.sol";
```

Comment:
The directory in the interfaces package should match this one, so it ought to be called `pool-stable`, without the `-composable` suffix

Reply:
I had it that way, but the problem is (as documented) there are still things using the original StablePool (including the proportional joins/exits that we removed). Maybe review those things to see if they can be deleted? Or make pool-stable-composable to pool-stable, and rename the current pool-stable to pool-stable-legacy?